lib/plugins/aws/deploy/lib/cleanupS3Bucket.js

```diff
@@ -2,59 +2,35 @@
 
 const BbPromise = require('bluebird');
 const _ = require('lodash');
+const findAndGroupStacks = require('../utils/findAndGroupStacks');
+const s3ObjectsFromStacks = require('../utils/s3ObjectsFromStacks');
```

Comment:
I would propose to prefix this to e.g. `getS3ObjectsFromStacks` so that it's easier to understand what this utils function does.